### PR TITLE
Bootstrap Assignment_02

### PR DESCRIPTION
**Pull request by NikitaShivhare1**

### CODE REVIEW

File: Nikita/Bootstap/Bootstrap Assignment_02/assign.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+            crossorigin="anonymous"
+        ></script>
+        <script
+            src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
+            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+            crossorigin="anonymous"
+        ></script>
+    </body>
+    </html>
```
Comment (sourabh513) [inline Nikita/Bootstap/Bootstrap Assignment_02/assign.html:82]: add new line at the end of the file

**Review by sourabh513 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

File: Nikita/Bootstap/Bootstrap Assignment_02/assign.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            .jumbotron {
+                padding-top: 30px;
+                padding-bottom: 50px;
+                margin-bottom: 30px;
+                background-color: yellowgreen;
+                background-image: url(./image/offline.jpg);
+                color: black;
+            }
+            .jumbotron h1,
+            p {
```
Comment (shubhamdalal49) [inline Nikita/Bootstap/Bootstrap Assignment_02/assign.html:23]: selector should be in single line

File: Nikita/Bootstap/Bootstrap Assignment_02/assign.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            crossorigin="anonymous">
+        </script>
+        <script
+            src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
+            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+            crossorigin="anonymous">
+        </script>
+    </body>
+    </html>
+
```
Comment (shubhamdalal49) [inline Nikita/Bootstap/Bootstrap Assignment_02/assign.html:83]: remove extra line from the end of the code

**Review by shubhamdalal49 — APPROVED** [review]

File: Nikita/Bootstap/Bootstrap Assignment_02/assign.html (hunk 3 above)
Comment (Pawan1005) [inline Nikita/Bootstap/Bootstrap Assignment_02/assign.html]: remove extra lines

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: Nikita/Bootstap/Bootstrap Assignment_02/assign.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+            crossorigin="anonymous">
+        </script>
+        <script
+            src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
+            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+            crossorigin="anonymous">
+        </script>
+    </body>
+    </html>
```
Comment (Pawan1005) [inline Nikita/Bootstap/Bootstrap Assignment_02/assign.html]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Pull request merged**